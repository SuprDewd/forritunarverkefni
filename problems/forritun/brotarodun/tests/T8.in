11
264/236
963/691
176/433
642/246
681/29
365/393
797/795
598/133
921/216
979/993
75/799
