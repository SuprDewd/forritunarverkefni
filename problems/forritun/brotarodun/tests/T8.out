75/799
176/433
365/393
979/993
797/795
264/236
963/691
642/246
921/216
598/133
681/29
